workflow/qa-steps.md

```diff
@@ -5,6 +5,17 @@ description: Recommendations for common QA steps when reviewing PRs
 
 In this section you will find a list of steps to take when reviewing PRs, in order to catch unforeseen regressions.
 
+## General guidelines for quality reviews
+
+- Assume positive intent, we all have collective responsibility for the code we write - be nice.
+- Avoid expressing subjective opinions without reasoning. If you think something is bad, you need to explain _why_ you think it is bad.
+- Try not to increase the scope of a PR unnecessarily. If a reviewing a PR leads you to unrelated code smells, create a new issue.
+- Embrace the 'social coding' aspect of Github. If you spot a typo, rather than type a comment explaining where you found the typo and what the correction should be, [suggest the actual code change](https://help.github.com/articles/incorporating-feedback-in-your-pull-request/#applying-a-suggested-change).
+- As above, you can also push refactors and larger code changes back to a PR which the original author can then review.
+- If you engage in a code review, you now have a responsibility to work with the author to get it over the line. Keep an eye out for your feedback being actioned and schedule a re-review in a timely manner.
```

Comment:
I'm slightly worried about this point - I don't want anything to discourage people from jumping in and collaborating on PRs. If someone wants to jump into a PR to make one comment but that's all they've got time for, that seems valuable.

I'd rather we had powerful processed for pushing forward with PRs without relying on anyone in particular. For example, I tend to "dismiss" "changes requested" reviews once all the points are addressed. I think this is a good pattern. Another would be not to assume that because someone might have assigned themselves to a PR that you can't also look at it.

Or maybe we could formalise what "assigned" means, and say don't assign yourself unless you can commit. What do you think?

Reply:
I don't disagree with any of the above - my point here is that PRs shouldn't languish because it's waiting for a reviewer to return to it.

Reply:
Great. So how about simply:

```suggestion
- If you engage in a code review, try to keep an eye out for your feedback being actioned so you can respond, to avoid blocking progress.
```